README.md

```diff
@@ -7,25 +7,39 @@ This dbt package contains models, macros, seeds, and tests for [FullStory](https
 | sessions | Session-level aggregations, including event counts broken down by type, location and device information, duration, FullStory session replay links, etc.
 | users | User-level aggregations, including email addresses, location and device information, session counts, etc.
 
+## Vars
+| var | description |
+| - | - |
+| fullstory_events_database | The database where your raw events table lives. |
+| fullstory_events_schema | The schema inside of your database where your raw events table lives. |
+| fullstory_events_table | The name of the table inside your schema where your raw events table lives. |
```

Comment:
nit: maybe change raw -> FullStory? For example, "where your FullStory events table lives"? Internally, raw is synonymous with the staging table that we use. Externally, I'm unsure what "raw" means to package consumers. Again, very minor.